helm_deploy/values-prod.yaml

```diff
@@ -24,3 +24,7 @@ generic-prometheus-alerts:
     - "book-a-prison-visit-prod-hmpps_prison_visits_event_dlq"
   sqsAlertsOldestThreshold: 1
   sqsAlertsTotalMessagesThreshold: 1
+
+hmpps:
```

Comment:
should this be HMMPS_SQS_ENABLED and be part of env (line 19)?

Reply:
Yeha you right, should be in env...